src/HtmlGenerator/Pass1-Generation/XmlSupport.cs

```diff
@@ -30,19 +30,10 @@ public void Generate(string sourceXmlFilePath, string destinationHtmlFilePath, s
             this.destinationHtmlFilePath = destinationHtmlFilePath;
 
             sourceText = File.ReadAllText(sourceXmlFilePath);
-
-            string html = GetHtml(sourceXmlFilePath, destinationHtmlFilePath, displayName);
-
-            var folder = Path.GetDirectoryName(destinationHtmlFilePath);
-            Directory.CreateDirectory(folder);
-            File.WriteAllText(destinationHtmlFilePath, html);
-        }
-
-        private string GetHtml(string sourceXmlFilePath, string destinationHtmlFilePath, string displayName)
-        {
             var lines = File.ReadAllLines(sourceXmlFilePath);
             lineLengths = sourceText.GetLineLengths();
             var lineCount = lines.Length;
+            var root = Parser.ParseText(sourceText);

```

Comment:
move the Parser call closer to usage, within the check for large files

Reply:
This wasn't an intentional change. I'm no longer touching this part of the file.